### PR TITLE
Chapter 14

### PR DESCRIPTION
**Pull request by starocketeer**

### CODE REVIEW

**Comment by starocketeer** [comment]
Ready

File: app/views/users/_unfollow.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<%= form_for(current_user.active_relationships.find_by(followed_id: @user.id),
+            html: { method: :delete },
+            remote: true) do |f| %>
```
Comment (manhnt-0960) [inline app/views/users/_unfollow.html.erb]: let's check indent

File: app/views/relationships/create.js.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/unfollow')) %>");
```
Comment (manhnt-0960) [inline app/views/relationships/create.js.erb]:
```suggestion
$("#follow_form").html("<%= j render('users/unfollow') %>");
```

File: app/views/users/_unfollow.html.erb (hunk 1 above)
Comment (starocketeer) [reply app/views/users/_unfollow.html.erb]: Done

File: app/views/relationships/create.js.erb (hunk 2 above)
Comment (starocketeer) [reply app/views/relationships/create.js.erb]: Done

**Comment by starocketeer** [comment]
Ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**